ginivision/src/main/java/net/gini/android/vision/review/multipage/MultiPageReviewFragment.java

```diff
@@ -248,6 +248,10 @@ public void onPlusButtonClicked() {
         mThumbnailsRecycler.setItemAnimator(itemAnimator);
     }
 
+    private boolean shouldShowPlusButton() {
+        return mMultiPageDocument.getImportMethod() != Document.ImportMethod.OPEN_WITH;
```

Comment:
Not sure about this. What if the user wants to add a page after importing one/several images?

Reply:
When launching with "open with" the user selected images in an app and sent it to GVL and there is no method to add more pages because we go directly to the Review Screen. More images can be added from within the GVL only in the Camera Screen.

The other import method is "picker" which is means the user was in the Camera Screen and used the file picker so we can show the "+" button. Also when the import method is not set the user must have come from the Camera Screen.

Reply:
And there is no way to add the camera to the stack when that occurs?

Reply:
Not without a major overhaul.
![](https://m.popkey.co/a0791d/YNO1Q.gif)